app/javascript/angular/code/services/_sensors.js

```diff
@@ -83,24 +83,6 @@ export const sensors = (params, storage, heat, $http) => {
     },
     buildSensorId: function(sensor) {
       return buildSensorId(sensor);
-    },
-    fetchHeatLevels: function() {
-      this.fetchHeatLevelsForSensor(this.selected());
-    },
-
-    fetchHeatLevelsForSensor: function(sensor) {
-      if (!sensor) return;
-
-      const callback = data => {
-        storage.updateDefaults({ heat: heat.parse(data) });
-        params.update({ data: { heat: heat.parse(data) } });
-      };
-      $http
-        .get("/api/thresholds/" + sensor.sensor_name, {
-          params: { unit_symbol: sensor.unit_symbol },
-          cache: true
-        })
-        .success(callback);
```

Comment:
This was used to store heat levels in storage, but that info was never used as far as I can see, so this fetch must be useless.